docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -28,24 +28,48 @@ Read more about Clerk session tokens and how they work in [the guide on how Cler
-    | `per` | permissions | The names of the permissions the user has in the organization. | `read,manage` |
-    | `fpm` | feature-permission map | The mapping of features with permissions, where the value of the integer, when converted to binary, represents a bitmask where each bit's position corresponds to a permission in the `o.per` list, and where `0` = `not-allowed` and `1` = `allowed`. | `3,2` |
+    The **`o` claim**, or organization claim, is only included if the user is part of an [organization](/docs/organizations/overview) and that organization is [active](!active-organization). Its value is an object that contains the following properties:
+
+    | Claim | Abbreviation expanded | Description | Example | |
+    | - | - | - | - | - |
+    | `id` | ID | The ID of the organization. | `org_123` | |
+    | `slg` | slug | The slug of the organization. | `org-slug` | |
+    | `rol` | role | The role of the user in the organization, without the `org:` prefix. | `admin` | |
+    | `per` | permissions | The names of the permissions the user has in the organization. | `read,manage` | |
+    | `fpm` | feature-permission map | feature-permission map | The mapping of features with permissions. [Learn how this value is constructed](#decode-o-fpm-manually). | `3,2` |
 
     > [!WARNING]
     > The organization claims above are intentionally designed to be as compact as possible to keep JWT tokens small.
     > As a result, they can be difficult to decode manually. We strongly recommend using one of our SDKs that support API version [2025-04-10](/docs/versioning/available-versions#2025-04-10) to handle decoding reliably.
 
+    #### Decode `o.fpm` manually
+
+    For those that need to decode the `o.fpm` claim manually, here is how the claim value is computed.
+    The value is a list of comma-separated integers where the position of the integer corresponds to a feature in the same position in the `fea` claim.
+    Each integer, when converted to binary, represents a bitmask where each bit's position corresponds to a permission in the `o.per` list, where `0` = `not-allowed` and `1` = `allowed`.
+
+    #### Example
+
+    Consider a user with an active organization has role `admin`. This role has the following feature permissions: `dashboard:read`, `dashboard:manage`, `teams:read`. Therefore, the user's claims are as follows:
+
+    - `fea` claim has value `o:dashboard,o:teams`
+    - `o.per` claim has value `manage,read`
+    - `o.fpm` claim has value `3,2`
+
+    Now, let's manually decode the `o.fpm` claim.
+
+    The first integer `3` represents the feature-permission mapping for the `dashboard` feature. Since `3` is `11` in binary, both permissions are allowed (`dashboard:read` and `dashboard:manage`).
+
+    The second integer `2` represents the feature-permission mapping for the `teams` feature. Since `2` is `10` in binary, the _second_ permission is allowed (`teams:read`) because the second bit from the right is `1`.
```

Comment:
@alexisintech I added "because the second bit from the right is `1`" at the end, let me know if you think that is helping here